packages/babel-parser/src/parser/statement.js

```diff
@@ -1585,6 +1587,18 @@ export default class StatementParser extends ExpressionParser {
       }
 
       this.checkGetterSetterParams(publicMethod);
+    } else if (
+      isContextual &&
+      key.name === "accessor" &&
+      this.hasPlugin("simpleAutoAccessors") &&
```

Comment:
This check can be moved to _inside_ the `if`, using `expectPlugin` rather than `hasPlugin` to report better errors ("missing plugin" vs "missing semicolon" probably)

Reply:
Please also add a new test case in `packages/babel-parser/test/fixtures/experimental/_no-plugin`.